src/main/java/org/tron/core/capsule/StorageRowCapsule.java

```diff
@@ -48,6 +48,12 @@ public StorageRowCapsule(byte[] rowKey, byte[] rowValue) {
     markDirty();
   }
 
+  public StorageRowCapsule(StorageRowCapsule rowCapsule) {
+    this.rowKey = rowCapsule.getRowKey().clone();
+    this.rowValue = rowCapsule.getRowValue().clone();
+    this.dirty = rowCapsule.isDirty();
+  }
+
   public StorageRowCapsule(byte[] rowValue) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6rASgN4uCclo1hE&open=AWbJU6rASgN4uCclo1hE)

[ISSUE_KEY]: AWbJU6rASgN4uCclo1hE